src/_base/harness/scripts/mutagen.sh

```diff
@@ -1,31 +1,54 @@
 #!/usr/bin/env bash
 
-if [ $# -ne 1 ]; then
-	echo "This script supports only one parameter"
-	exit 1
+if [ "$#" -ne 1 ]; then
+    echo "This script supports only one parameter"
+    exit 1
 fi
 
-COMMAND=$1
+COMMAND="$1"
+PATH="$PATH:./.my127ws/utilities/mutagen/"
 
 install_mutagen()
 {
-    if ! command -v mutagen > /dev/null 2>&1; then
-      passthru brew install mutagen-io/mutagen/mutagen
+    if command -v mutagen > /dev/null 2>&1; then
+        return 0
     fi
+
+    if command -v sw_vers > /dev/null 2>&1 && sw_vers | grep -q Mac; then
+        if command -v brew > /dev/null 2>&1; then
+            passthru brew install mutagen-io/mutagen/mutagen
+        else
+            mkdir -p .my127ws/utilities/mutagen/
+            run curl -L -q -sS -f https://github.com/mutagen-io/mutagen/releases/download/v0.11.2/mutagen_darwin_amd64_v0.11.2.tar.gz -o .my127ws/utilities/mutagen/mutagen.tar.gz
```

Comment:
Is there a way to get latest release tag, so we always download latest version? 🤔